pkg/volume/volume_linux.go

```diff
@@ -28,8 +28,9 @@ import (
 )
 
 const (
-	rwMask = os.FileMode(0660)
-	roMask = os.FileMode(0440)
+	rwMask   = os.FileMode(0660)
+	roMask   = os.FileMode(0440)
+	execMask = os.FileMode(0110)
```

Comment:
Mask is a misnomer here, or there's a bug. If I'm reading it correctly, these permissions are strictly additive with whatever permissions are already on the files?

Reply:
it's a "mask" in that we think of perms as a bitmask not as an AND mask

Reply:
I think you're fine